docs/topics/migrations.txt

```diff
@@ -333,6 +336,12 @@ Note that this only works given two things:
   that your database doesn't match your models, you'll just get errors when
   migrations try to modify those tables.
 
+Note: if you have upgraded from Django 1.7 or before, and get
+'foreign key constraint' errors (for example, when you run manage.py test),
```

Comment:
I think the solution is to add migrations for the unmigrated app (as noted in ticket #24311, 1.7 would fail silently, while 1.8 fails loudly). This could be mentioned in the 1.8 release notes. 

The relevant section of the docs is https://docs.djangoproject.com/en/1.8/topics/migrations/#dependencies